astropy/modeling/functional_models.py

```diff
@@ -1123,6 +1167,18 @@ def __init__(self, amplitude=amplitude.default, x_0=x_0.default,
             amplitude=amplitude, x_0=x_0, y_0=y_0, r_in=r_in, width=width,
             **kwargs)
 
+    def bounding_box_default(self):
```

Comment:
My point was that testing on the sum is kind of redundant, when the resulting arrays already exactly agree.
And testing every pixel in an array is much more sensitive, then checking the sum only. But what I see now, is that one could add a test, that checks, whether the default values for the bboxes make sense (and that's what you probably had in mind...) and this could be done by checking the summed flux.